demo/twenty_news/build.gradle

```diff
@@ -17,15 +17,10 @@ dependencies {
   compile 'org.slf4j:slf4j-simple:1.7.7'
 }
 
-compileScala {
```

Comment:
Side discussion, can we switch to gradle wrapper? This could ensure that everyone's using exactly same gradle to compile.